ccd-definition/CaseEventToFields/DefendantResponseAdmitPartLRspec.json

```diff
@@ -241,5 +241,46 @@
     "PageFieldDisplayOrder": 2,
     "DisplayContext": "COMPLEX",
     "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
+    "CaseFieldID": "respondent1DQIncomeAndExpenseLabel",
+    "PageID": "DefendantIncomeExpenses",
+    "PageDisplayOrder": 20,
+    "PageFieldDisplayOrder": 1,
+    "PageShowCondition": "respondent1ClaimResponseTypeForSpec=\"PART_ADMISSION\"",
+    "DisplayContext": "READONLY",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
+    "CaseFieldID": "respondent1DQCarerAllowanceCredit",
+    "PageID": "DefendantIncomeExpenses",
+    "PageDisplayOrder": 20,
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "MANDATORY",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
+    "CaseFieldID": "respondent1DQRecurringIncome",
+    "PageID": "DefendantIncomeExpenses",
+    "PageDisplayOrder": 20,
+    "PageFieldDisplayOrder": 3,
+    "DisplayContext": "COMPLEX",
```

Comment:
I think Mandatory will work here because we are not defining 'mandatory' attribute on individual elements of the field (collection). Or is capturing income and expense mandatory on this screen(i.e. can a person have no income or expense)?

Reply:
When the field is a collection, mandatory sets its minimum to 1 element. That's why I used complex instead